psi-sample/README.md

```diff
@@ -0,0 +1,55 @@
+## PSI - Private Set Intersection Sample
+
+This is a simple app using the Conclave API. It is licensed under the Apache 2 license, and therefore you may 
+copy/paste it to act as the basis of your own commercial or open source apps.
+
+Using this sample will see how Conclave (based on Intel SGX) can be a new tool to solve the private set intersection problem (PSI)
+
+# What is Private Set Intersection Problem (PSI) ?
+
+PSI refers to the problem of determining the common elements from the intersection of two sets without leaking or disclosing any 
+additional information of the remaining elements of the either sets.
+
+# Use Case : Measuring ad conversion rates
+
+Measuring ad conversion rates is done by comparing the list of people who have seen an ad with those who have completed a transaction. 
+These lists are held by the advertiser (say, Google or Facebook), and by merchants, respectively. 
+It is often possible to identify users on both ends, using identifiers such as credit card numbers, email addresses, etc. 
+A simple solution, which ignores privacy, is for one side to disclose its list of customers to the other side, which then computes the necessary statistics. 
+
+# Parties involved in the sample
```

Comment:
Maybe it would be good to add a note on the best way to read the sample, e.g.
* focus on Enclave, Host, and Client classes
* what order to read them in?

Reply:
Have updated the Readme to make it more clear.